### PR TITLE
EMSUSD-645 - As a user, I'd like to add a relative file dependency on a anonymous layer

### PR DESCRIPTION
**Pull request by vlasovi**
Postponed processing of relative asset paths assigned to anonymous layers

### CODE REVIEW

File: lib/mayaUsd/utils/utilFileSystem.h [hunk 1]
```diff
@@ -143,6 +143,21 @@ void unmarkPathAsPostponedRelative(
     const PXR_NS::SdfLayerHandle& layer,
     const std::string&            contentPath);
 
+/*! \brief Performs handling of a USD asset path attribute that maybe relative to a layer.
+Whether the file path is required to be relative is defined by the primvar which name is supplied
+through 'primVarName' variable. If the relative file path is required and the layer is saved on disk
```
Comment (pierrebai-adsk) [inline lib/mayaUsd/utils/utilFileSystem.h:148]: Is it not an 'optionVar' name and not a 'primVar' name?

**Review by pierrebai-adsk — CHANGES_REQUESTED** [review]
Ingeniously done, just a small requested change.

File: lib/mayaUsd/utils/utilFileSystem.h (hunk 1 above)
Comment (vlasovi) [reply lib/mayaUsd/utils/utilFileSystem.h:148]: Yeah, you are right. I'll fix it

**Review by pierrebai-adsk — APPROVED** [review]
LGTM, good job

**Pull request merged**